docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -365,7 +365,7 @@ Release
         export VERSION=0.1.2
 
```

Comment:
I will put some intervening text in.

It is not possible - `|latest-installable|` on `master` (which doc-dev is built from, doc-dev is used for the release) uses the latest tag. When the docs are built the tag will be that of the previous release.